app/controllers/v0/id_card_attributes_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'vic/url_helper'
+require 'vic/id_card_attribute_error'
+
+module V0
+  class IdCardAttributesController < ApplicationController
+    before_action :authorize
+
+    def show
+      id_attributes = IdCardAttributes.for_user(current_user)
+      vic_url = VIC::URLHelper.generate_url(id_attributes)
+      render json: { 'redirect' => vic_url }
```

Comment:
if all we're providing is a url, should the routes/controller name reflect that instead? I realize the url generated is based on these attributes, but if the action returns a url, i think the name ought to reflect that instead, otherwise just return the attributes maybe and let the F/E build the URL?

Reply:
How about route `id_card/request_url`? 


I would rather keep all the logic in one place rather than building the URL on the frontend and needing to add more tests there.